src/lib/web-publisher-status.js

```diff
@@ -5,6 +5,7 @@ class PublisherStatus {
     const client = await database_pool.connect();
     const publisherSqlQuery = `
      SELECT
+        publisher.id as "csPublisherId",
```

Comment:
We're not live so it doesn't matter, but noting that if we were I think this would count as a breaking change and we'd have to make sure we release this and the status change P.R. at the same time? Just worth bearing in mind for later releases if so

Reply:
We didn't expose this value previously so there is no change to break. But yes, the more things we add the more we need to be careful to not make breaking changes between the conform serverices and the status page.